fleet/api/openapi.yaml

```diff
@@ -103,6 +103,125 @@ paths:
           description: Missing or invalid access token provided.
         '500':
           $ref: "#/components/responses/ServiceErrorRes"
+  /agents:
+    parameters:
+      - $ref: "#/components/parameters/Authorization"
+    get:
+      summary: 'List current Agents'
+      operationId: listAgents
+      tags:
+        - agents
+      parameters:
+        - $ref: "#/components/parameters/Limit"
+        - $ref: "#/components/parameters/Offset"
+        - $ref: "#/components/parameters/Name"
+        - $ref: "#/components/parameters/Order"
+        - $ref: "#/components/parameters/Direction"
+      responses:
+        '200':
+          $ref: "#/components/responses/AgentPageRes"
+        '400':
+          description: Failed due to malformed query parameters.
+        '401':
+          description: Missing or invalid access token provided.
+        '404':
+          description: A non-existent entity request.
+        '500':
+          $ref: "#/components/responses/ServiceErrorRes"
+    post:
+      summary: 'Create a new Agent'
```

Comment:
this is missing parameters/Authorization i think?

Reply:
Done, i was trying to use the security schema, i forgot to rollback.